client/src/views/UploadView.vue

```diff
@@ -65,6 +66,7 @@ export default {
         this.getValidationOverrideLevel(),
         this.updateStatus
       );
+      store.set_plan(jsonFileRes, this.getValidationOverrideLevel());
```

Comment:
Probably not urgent, but is there a way for `plot` to run as a "change callback" when `plan` is modified in the store?

Reply:
We don't have that directly. But, a change in plan in either the CodeView or the UploadView, and then clicking the `Generate` button will always call the `plot` function. 

It might be a UX choice, if we implement a change callback, then without even clicking the `Generate` button, the plot might be updated.

Also, we can have another change. The plot function is not required to run if there is no change in the `plan`. We may restrict the validation, and further plot if we receive the same plan that is already stored, that way we may save some back-end API calls to the validator, and now storing the plan in the database.